compiler/shared_ast/var.ml

```diff
@@ -21,14 +21,15 @@ open Definitions
 (** This module provides types and helpers for Bindlib variables on the [gexpr]
     type *)
 
-type 'e t = ('a, 't) naked_gexpr Bindlib.var constraint 'e = ('a any, 't) gexpr
+type 'e t = ('a any, 't) naked_gexpr Bindlib.var constraint 'e = ('a, 't) gexpr
 
 type 'e vars = ('a, 't) naked_gexpr Bindlib.mvar
   constraint 'e = ('a any, 't) gexpr
 
 let make (name : string) : 'e t = Bindlib.new_var (fun x -> EVar x) name
 let compare = Bindlib.compare_vars
 let eq = Bindlib.eq_vars
+let translate_mark v = Bindlib.copy_var v (fun x -> EVar x) (Bindlib.name_of v)
```

Comment:
What is the difference with `translate` just below?

Reply:
The implementation happens to be the same, but the difference is in the  `mli`: this one has a more restricted type.

But you're right that just adds noise, I'll remove it.